packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -8,27 +8,51 @@ import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 import { Editor, Transforms, Element as SlateElement } from 'slate';
 import { useSlate } from 'slate-react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 import { useBlocksStore } from '../hooks/useBlocksStore';
 import { useModifiersStore } from '../hooks/useModifiersStore';
 
+const ToolbarWrapper = styled(Flex)`
+  ${(props) =>
+    props.disabled &&
+    css`
+      cursor: not-allowed;
```

Comment:
this cursor looks actually different from disabled editor or select, I think it should be the same everywhere.

Reply:
These are the design specs after checking with @lucasboilly:

- the entire toolbar should have the 🚫 cursor
- the body of the editor should have the normal default pointer to make it easy for users to select things. we just need to prevent interactions that modify content

The issue currently in my PR is that the select stands out a bit, having a different background color and cursor when disabled. But I think it's part of a larger issue of the Select from the design system not matching the one we have in the designs. So I added details to the [ticket](https://strapi-inc.atlassian.net/browse/CS-271) we already have to customize its appearance to handle this too.